### PR TITLE
doc: matter: certification templates section

### PR DESCRIPTION
**Pull request by greg-fer**
Added a section about certification templates for Matter. KRKNWK-15725.

Signed-off-by: Grzegorz Ferenc <Grzegorz.Ferenc@nordicsemi.no>

### CODE REVIEW

File: doc/nrf/ug_matter_device_certification.rst [hunk 1]
```diff
@@ -253,7 +261,15 @@ You can visit the following pages on Nordic Semiconductor Infocenter to check th
 * `nRF5340 DK Compatibility Matrix`_
 * `nRF52840 DK Compatibility Matrix`_
 
+Certification templates
```
Comment (LuDuda) [inline doc/nrf/ug_matter_device_certification.rst:264]:
```suggestion
Certification documents templates
```

**Review by LuDuda — APPROVED** [review]
Looks great!

File: doc/nrf/ug_matter_device_certification.rst [hunk 2]
```diff
@@ -243,6 +243,14 @@ Matter dependent certification inheritance
 If your product uses qualified Bluetooth stack and certified Thread libraries provided as part of the |NCS|, you can *inherit* certification from Nordic Semiconductor, provided that you do not introduce any changes to these stacks.
 In practice, this means reusing Nordic Semiconductor's certification identifiers, which were obtained as a result of the official certification procedures.
 
+.. note::
+   The inheritance is granted by the related SDO after you join the organization and apply for certification by inheritance.
```
Comment (doublemis1) [inline doc/nrf/ug_matter_device_certification.rst:247]: certification by inheritance is an application specific for the Thread Group
the Wi-Fi SDO has a QuickTrack or Derative certification option,
the Bluetooth QDID can be obtained by Bluetooth Qualification Process with
“No Required Testing”.

**Review by doublemis1 — APPROVED** [review]
LGTM

**Pull request merged**